### DISCUSSION TITLE
Help in the beginning

### DISCUSSION THREAD

**Discussion by TestGit-2023** [def-studio/telegraph#450] [Q&A]
Hello. I am a novice developer. I decided to explore the world of telegram bots. The choice fell on your project. I figured out the callback buttons, the command processing... And I'm stuck on how do I do this: I want to ask the user for their interests. That is, first there is a check whether his interests are filled in the database, and if not, the bot sends a message "Specify your interests". After the user has entered (text) his interests - how do I send his text to a specific handler? That is, the public function addInterest()

**Comment by erkinov-wtf** [comment]
if you can understand russian there's best video(so far) in youtube. guy explains it vey simply. link - https://www.youtube.com/watch?v=1J8l3pCc8p8&list=WL&index=1

**Comment by TestGit-2023** [comment]
> if you can understand russian there's best video(so far) in youtube. guy explains it vey simply. link - https://www.youtube.com/watch?v=1J8l3pCc8p8&list=WL&index=1

Yes, I understand Russian and I watched this video)
The fact is that the author tells how to use the buttons. For example:
`->keyboard(Keyboard::make()->buttons([
                Button::make('Delete')->action('delete')->param('id', '42'),
                Button::make('open')->url('https://test.it'),
               Button::make('Web App')->webApp('https://web-app.test.it'),
               Button::make('Login Url')->loginUrl('https://loginUrl.test.it'),
           ]))->send();`

I need something like this
`$this->chat->message('Enter text')->action('myfunction')->send();`
And after the user enters something, then I will receive his response in the function myfunction

**Comment by erkinov-wtf** [comment]
yeah i also need to learn the thing you said

**Comment by TestGit-2023** [comment]
> yeah i also need to learn the thing you said

If there are results, please report in this thread. Thank you)
